torchci/clickhouse_queries/torchbench_list_userbenchmarks/query.sql

```diff
@@ -1,6 +1,5 @@
--- !!! Query is not converted to CH syntax yet.  Delete this line when it gets converted
 SELECT DISTINCT
     name
 FROM
-    torchbench."torchbench-userbenchmark"
-WHERE name is not NULL
\ No newline at end of file
+    benchmark.torchbench_userbenchmark
+WHERE name is not NULL
```

Comment:
Depending on the type of name, name is not NULL might always be true

Reply:
I'm removing the WHERE.